.yarnrc.yml

```diff
@@ -0,0 +1,2 @@
+nodeLinker: node-modules
+enableGlobalCache: true
```

Comment:
Adding `compressionLevel: 0` here will make installs much faster, because the cache will be in uncompressed zip files and Yarn will not spend time and CPU on compression and decompression.